console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/GwtKapuaErrorCode.java

```diff
@@ -41,5 +41,7 @@
     ENTITY_ALREADY_EXIST_IN_ANOTHER_ACCOUNT,
     SELF_LIMIT_EXCEEDED_IN_CONFIG,
     MAX_NUMBER_OF_ITEMS_REACHED,
-    OPERATION_NOT_ALLOWED_ON_ADMIN_USER
+    OPERATION_NOT_ALLOWED_ON_ADMIN_USER,
+    RETRY_AND_CRON_BOTH_SELECTED,
```

Comment:
Very specific error code in enum that is meant for general error codes.

Reply:
@Coduz Do you have maybe some suggestion for this part?